packages/main/src/SegmentedButton.ts

```diff
@@ -289,12 +328,26 @@ class SegmentedButton extends UI5Element {
 	 * @readonly
 	 * @type {sap.ui.webc.main.ISegmentedButtonItem}
 	 * @name sap.ui.webc.main.SegmentedButton.prototype.selectedItem
+	 * @deprecated As of 1.14.0. This method is not guaranteed in future releases.
+	 * Please use the <code>pressedItems</code> property instead.
 	 * @public
 	 */
 	get selectedItem() {
 		return this._selectedItem;
 	}
 
+	/**
+	 * Retrieves an array of <code>SegmentedButtonItem</code>'s.
+	 * @readonly
+	 * @name sap.ui.webc.main.SegmentedButton.prototype.selectedItem
```

Comment:
the name is wrong
@name sap.ui.webc.main.SegmentedButton.prototype.selectedItem

Reply:
Done